Bottom: f743cceab57225c73f69d9e728199f1904a8f819
Top:    ecbb21b74f7c18bfc6088f0c1d5d1dcbd96c25ca
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 11:20:13 +0000

Refresh of encapsulate-the-find-setup-in-

---

diff --git a/src/find.c b/src/find.c
index 31faa89..0413d15 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3172,7 +3172,7 @@ LookupUnusedPins (FILE * FP)
     gui->beep ();
   FreeConnectionLookupMemory ();
   IncrementUndoSerialNumber ();
-  setip.user = false;
+  setup.user = false;
   Draw ();
 }
